messaging-activemq/src/main/resources/org/wildfly/extension/messaging/activemq/LocalDescriptions.properties

```diff
@@ -78,8 +78,11 @@ broadcast-group.group-address.deprecated=Use instead a socket-binding to specify
 broadcast-group.group-address=Deprecated. The multicast address to which the data will be broadcast. It is a class D IP address in the range 224.0.0.0 to 239.255.255.255, inclusive. The address 224.0.0.0 is reserved and is not available for use.
 broadcast-group.group-port.deprecated=Use instead a socket-binding to specify the broadcast group's group port.
 broadcast-group.group-port=Deprecated. The UDP port number used for broadcasting.
-broadcast-group.jgroups-channel=The name used by a JGroups channel to join a cluster.
-broadcast-group.jgroups-stack=The name of a stack defined in the org.jboss.as.clustering.jgroups subsystem that is used to form a cluster.
+broadcast-group.jgroups-cluster=The logical cluster name.
+broadcast-group.jgroups-channel=References the name of a JGroups channel.  If undefined, the default channel will be used.
+broadcast-group.jgroups-channel.deprecated=Deprecated. Use jgroups-cluster instead.
+broadcast-group.jgroups-stack=References the name of a JGroups channel factory.
+broadcast-group.jgroups-stack.deprecated=Deprecated. Use jgroups-channel-ref instead.
```

Comment:
I suppose you mean to use `jgroups-channel` instead

Reply:
Correct. Will fix.